functions/go/set-namespace/transformer/namespace.go

```diff
@@ -145,13 +152,18 @@ func FindAllNamespaces(objects []*fn.KubeObject) ([]string, map[string]int) {
 }
 
 // ReplaceNamespace iterates the `objects` to replace the `OldNs` with `newNs` on namespace field.
-func ReplaceNamespace(objects []*fn.KubeObject, oldNs, newNs string) {
+func ReplaceNamespace(objects []*fn.KubeObject, oldNs, newNs string, count *int) {
+	*count = 0
```

Comment:
Is this line necessary when you have `count := new(int)` before passing it in?
Why don't we simply return this value instead of passing in an int pointer?

Reply:
good catch! 

"Why don't we simply return this value instead of passing in an int pointer?" --> because `replaceNamespace` should not expect a return with an integer telling how many fields are updated. The function name should expect an error or silence.